doc/rst/source/events_common.rst_

```diff
@@ -204,6 +204,22 @@ Optional Arguments
 
 .. include:: explain_-XY.rst_
 
+.. _-Z:
+
+**-Z**\ "custom command"
+    Place symbols generated by either :doc:`/supplements/seis/coupe`,
+    :doc:`/supplements/seis/meca`, or :doc:`/supplements/geodesy/velo`.
+    The quoted custom command must start with one of these module names, then
+    supply all the module-specific options that the selected module requires to
+    normally define a plot, excluding any of **-C**, **-G**, **-J**, **-N**, **-R**, and
+    **-W** (those options may be required by **events** though).  Also, you may
+    not use the **-I** or **-t** options since these will be set automatically
+    as part of the variations imposed by **-Mi** and **-Mt**. **Note**: If
+    you are running a simple classic command then you must use classic module names.
+    As an example, the custom command to plot a beachball may be **-Z**\ "meca -Sa5c+f0",
+    while displaying a crossection of one may require the more elaborate command
+    **-Z**\ "coupe -Q -L -Sc3c -Ab128/11/120/250/90/400/0/100+f -Fa0.1i/cc".
```

Comment:
```suggestion
    as part of the variations imposed by **-Mi** and **-Mt**.  As an example, the
    custom command to plot a beachball may be **-Z**\ "meca -Sa5c+f0", while
    displaying a crossection of one may require the more elaborate command
    **-Z**\ "coupe -Q -L -Sc3c -Ab128/11/120/250/90/400/0/100+f -Fa0.1i/cc".
    **Note**: If you are running a simple classic command then you must use
    classic module names.
```
It makes more sense to me to but the classic command note after the modern mode example.